doc/conf.py

```diff
@@ -1,157 +1,12 @@
-import os
-import sys
-import time
+# Obtain shared config values
+import os, sys
 
-# Configuration file for the Sphinx documentation builder.
+sys.path.append(os.path.abspath(".."))
```

Comment:
Better be relative to `os.path.dirname(__file__)`, I do not like to have code working dependent on start directory. I know it's more tedious to write, but it avoids issues later on.